data/users/externaloidc.py

```diff
@@ -150,7 +152,7 @@ def sync_oidc_groups(self, user_groups, user_obj):
                     logger.exception(
                         f"External OIDC Group Sync: Exception occurred when adding user: {user_obj.username} to quay team: {team_synced.team} as {err}"
                     )
-                except UserAlreadyInTeam:
+                except (IntegrityError, UserAlreadyInTeam):
```

Comment:
Instead of catching the integrity error do we want to check if this entry exists instead? That would remove unnecessary errors from logs and would be easier to understand for the future